modules/local/bam_remove_orphans.nf

```diff
@@ -17,6 +14,9 @@ process BAM_REMOVE_ORPHANS {
     tuple val(meta), path("${prefix}.bam"), emit: bam
     path "versions.yml"                   , emit: versions
 
+    when:
```

Comment:
Why adding the `when` to the local modules if it is not used? I see that this way they have the same structure as nf-core modules but since they are local and we are not using it so I won't add it.

Reply:
I thought the linter complained about this in local modules? That's why I added them here too. Haven't checked. Either way, no harm right?